vmi.py

```diff
@@ -0,0 +1,280 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.14.1
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# %%
+import pandas as pd
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+import seaborn as sns
+
+# %%
+islaidos=pd.read_csv("Gyventojuislaidos.csv")
+
+# %%
+#Išlaidos#
+
+# %%
+islaidos.info() #paklausti del strukturoje esanciu beveik visu string
+
+# %%
+islaidos.nunique()
+
+# %%
+islaidos.head(10)
+
+# %%
+islaidos['suma'].nlargest(n=10) #10 didziausiu islaidu
+#prideti ir rusi/savivaldybe
+#is apraso "Išlaidų suma esanti virš 600000 eurų yra fiksuojama 600000 verte"
+
+# %%
+islaidos['suma'].nsmallest(n=10) #KAIP IMTI BE NULIU IR PRIDETI RUSI ARBA ISTRINTI NULINES
+
+# %%
+#------------RUSYS----------------#
+
+# %%
+islaidos_rusys=islaidos.groupby(['ir_kodas', 'pavadinimas']).sum(['suma']).drop(['metai','sav_kodas'],axis=1)
+islaidos_rusys #pagal islaidu rusies kodus susumuota islaidu sumos
+
+# %%
+mpl.rc('figure', figsize=(10,10))
+islaidos_rusys.reset_index().plot(x='ir_kodas', y='suma', c = 'red', marker = 'o',  markersize=7, grid = True).legend(labels = ['Suma'])
```

Comment:
Labai ilgos eilutės, reikėtų laužyti:

```suggestion
(
    islaidos_rusys.
    reset_index().
    plot(
        x='ir_kodas',
        y='suma',
        c='red',
        marker='o',
        markersize=7,
        grid=True,
    ).
    legend(labels=['Suma'])
)
```

Reply:
Būtų gerai sužinoti, ką reiškia kiekvienas išlaidų rūšies kodas, gal būti kodų reikšmės turėtu būti pateiktos struktūros apraše.